_sources/SimplePythonData/FunctionCalls.rst

```diff
@@ -0,0 +1,102 @@
+..  Copyright (C)  Brad Miller, David Ranum, Jeffrey Elkner, Peter Wentworth, Allen B. Downey, Chris
+    Meyers, and Dario Mitchell.  Permission is granted to copy, distribute
+    and/or modify this document under the terms of the GNU Free Documentation
+    License, Version 1.3 or any later version published by the Free Software
+    Foundation; with Invariant Sections being Forward, Prefaces, and
+    Contributor List, no Front-Cover Texts, and no Back-Cover Texts.  A copy of
+    the license is included in the section entitled "GNU Free Documentation
+    License".
+
+.. qnum::
+   :prefix: data-4-
+   :start: 1
+
+Function Calls
```

Comment:
This seemed a little out of the blue to me.  Why do function calls get their own subchapter in a chapter called, Simple Python Data.  I think there are two solutions:

1.  Change the chapter title to indicate that we are really giving them an overview of Python/programming languages.

2.  Emphasize that functions are just another data type...  This is super important but I think it is also a bit subtle for  this early??

Reply:
@presnick - would like your thoughts on this.

Reply:
I really like explicitly teaching them about function calls this early. We want to use them but not define them at this point in the course, and I think it's better to do it explicitly.

I think we should rename the chapter to "Expressions and Variables". Function calls are a natural component of expressions.

Reply:
How about Expressions, Statements and Variables